packages/turbo/ios/RNSessionModule.swift

```diff
@@ -30,6 +33,9 @@ class RNSessionModule: RCTEventEmitter {
     _ sessionHandle: NSString,
     resolver resolve: @escaping RCTPromiseResolveBlock,
     rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
+      if let index = supportedEventNames.firstIndex(of: "sessionMessage" + (sessionHandle as String)) {
+        supportedEventNames.remove(at: index)
+      }
```

Comment:
You might be wondering if that's a good approach... Well, from what I tested, `removeSession` is called rarely, so it's better to do `O(n)` search rather than converting `session.keys` every time `supportedEvents` is called (and `supportedEvents` method is called every time website sends an event to native.